package.json

```diff
@@ -35,7 +35,6 @@
   },
   "homepage": "https://github.com/asyncapi/website#readme",
   "dependencies": {
-    "@asyncapi/modelina": "^0.49.0",
```

Comment:
Kindly add the changes of `package-lock.json` file too in the npm version of `v8.8.0`.